payments/credit-account/redirect.md

```diff
@@ -0,0 +1,386 @@
+        "intent": "Authorization",
+        "currency": "SEK",
+        "amount": 1500,
+        "remainingCaptureAmount": 1500,
+        "remainingCancellationAmount": 1500,
+        "remainingReversalAmount": 0,
+        "description": "Test Purchase",
+        "payerReference": "AB1234",
+        "initiatingSystemUserAgent": "PostmanRuntime/3.0.1",
+        "userAgent": "Mozilla/5.0...",
+        "language": "sv-SE",
+        "prices": {
+            "id": "/psp/creditaccount/payments/{{ page.paymentId }}/prices"
+        },
+        "transactions": {
+            "id": "/psp/ccreditaccount/payments/{{ page.paymentId }}/transactions"
+        },
+        "authorizations": {
+            "id": "/psp/creditaccount/payments/{{ page.paymentId }}/authorizations"
+        },
+        "captures": {
+            "id": "/psp/creditaccount/payments/{{ page.paymentId }}/captures"
+        },
+        "reversals": {
+            "id": "/psp/creditaccount/payments/{{ page.paymentId }}/reversals"
+        },
+        "cancellations": {
+            "id": "/psp/creditaccount/payments/{{ page.paymentId }}/cancellations"
+        },
+        "payeeInfo": {
+            "id": "/psp/creditaccount/payments/{{ page.paymentId }}/payeeInfo"
+        },
+        "urls": {
+            "id": "/psp/creditaccount/payments/{{ page.paymentId }}/urls"
+        },
+        "settings": {
+            "id": "/psp/creditaccount/payments/{{ page.paymentId }}/settings"
+        },
+        "approvedLegalAddress": {
+            "id": "/psp/creditaccount/payments/{{ page.paymentId }}/approvedlegaladdress"
+        },
+        "maskedApprovedLegalAddress": {
+            "id": "/psp/creditaccount/payments/{{ page.paymentId }}/maskedapprovedlegaladdress"
+        }
+    },
+    "approvedLegalAddress": {
+        "id": "/psp/creditaccount/payments/{{ page.paymentId }}/approvedlegaladdress"
+    },
+    "operations": [
+        {
+            "href": "{{ page.apiUrl }}/psp/creditaccount/payments/{{ page.paymentId }}/captures",
+            "rel": "create-capture",
+            "method": "POST"
+        },
+        {
+            "href": "{{ page.apiUrl }}/psp/creditaccount/payments/{{ page.paymentId }}/cancellations",
+            "rel": "create-cancel",
+            "method": "POST"
+        },
+        {
+            "href": "{{ page.apiUrl }}/psp/creditaccount/payments/{{ page.paymentId }}/approvedlegaladdress",
+            "rel": "create-approved-legal-address",
+            "method": "POST"
+        }
+    ]
+}
+```
+
+If a `GET` method is used from `paymentUI` with a `paymentToken`, the following
+operations can be returned, depending on state of the payment and the last
+transaction.
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "href": "https://example.com/cancelUrl",
+    "rel": "redirect-merchant-cancel",
+    "method": "GET"
+}
+{
+    "href": "https://example.com/completeUrl",
+    "rel": "redirect-merchant-complete",
```

Comment:
This operation isn't very valuable to the merchant.

Reply:
this is now deleted from this file